plutarch-testlib/goldens/rational.bench.golden

```diff
@@ -23,4 +23,4 @@ data.id.2 {"exBudgetCPU":16100,"exBudgetMemory":200,"scriptSizeBytes":12}
 data.id.11/3 {"exBudgetCPU":6235110,"exBudgetMemory":26827,"scriptSizeBytes":239}
 div by 0.1/0 {"exBudgetCPU":278933,"exBudgetMemory":136,"scriptSizeBytes":239}
 div by 0.recip 0 {"exBudgetCPU":187980,"exBudgetMemory":134,"scriptSizeBytes":123}
-div by 0.1/(1-1) {"exBudgetCPU":4724786,"exBudgetMemory":20156,"scriptSizeBytes":286}
+div by 0.1/(1-1) {"exBudgetCPU":4825994,"exBudgetMemory":20158,"scriptSizeBytes":286}
```

Comment:
:crying_cat_face: Do we know why?

Reply:
I wish I did, but AFAICT, nothing changed in a way that would trigger _this specific case_.